src/brasil/gov/vlibrasnews/behaviors/interfaces.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+from brasil.gov.vlibrasnews import _
+from plone.autoform.interfaces import IFormFieldProvider
+from plone.supermodel import model
+from zope import schema
+from zope.interface import provider
+
+
+@provider(IFormFieldProvider)
+class IVLibrasNews(model.Schema):
+
+    """Enable VLibras News API behavior."""
+
+    model.fieldset('settings', fields=['video_url'])
+
+    video_url = schema.ASCIILine(
```

Comment:
let's keep ``schema.ASCIILine`` please, the validation don't accept empty string or ``None``